src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -128,6 +133,8 @@ private void handleOperatorList(RoutingContext rc) {
                 jo.put("created", o.getCreated());
                 jo.put("disabled", o.isDisabled());
                 jo.put("site_id", o.getSiteId());
+                jo.put("is_public_operator", o.isPublicOperator());
```

Comment:
I don't think having both `is_public_operator` and `is_private_operator` is a good idea. They can't be both false or both true, this is just error cases that now would need to be handled.  My preference would be to have a something like `operatorType` and that could have values of `public` and `private`. This would only have one error condition (the type being unknown/null) and would allow for extensibility if we have another operator type in the future (e.g. shared). This would also remove the need for the awkward `equals("true")` below.

Reply:
Changed as suggested